wallet/test/widget_testing/components/buttons/nft_buy_button_test.dart

```diff
@@ -1,28 +1,66 @@
+import 'dart:io';
 import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:get_it/get_it.dart';
+import 'package:mockito/mockito.dart';
+import 'package:provider/provider.dart';
 import 'package:pylons_wallet/generated/locale_keys.g.dart';
+import 'package:pylons_wallet/pages/home/currency_screen/model/ibc_coins.dart';
+import 'package:pylons_wallet/pages/purchase_item/purchase_item_screen.dart';
+import 'package:pylons_wallet/pages/purchase_item/purchase_item_view_model.dart';
+import 'package:pylons_wallet/pages/purchase_item/widgets/buy_nft_button.dart';
+import 'package:pylons_wallet/stores/wallet_store.dart';
+import 'package:pylons_wallet/utils/constants.dart';
 import '../../../mocks/mock_constants.dart';
+import '../../../mocks/purchase_item_view_model.mocks.dart';
 import '../../extension/size_extension.dart';
+import '../../../mocks/mock_wallet_store.dart';
 import 'package:easy_localization/easy_localization.dart';
-import 'package:pylons_wallet/pages/home/currency_screen/model/ibc_coins.dart';
-import 'package:pylons_wallet/pages/purchase_item/widgets/buy_nft_button.dart';
 
 void main() {
   TestWidgetsFlutterBinding.ensureInitialized();
+  late WalletsStore walletStore;
+  late PurchaseItemViewModel viewModel;
```

Comment:
`PurchaseItemViewModel viewModel =  MockPurchaseItemViewModel();`

Why not this?